src/containers/Page.jsx

```diff
@@ -73,6 +86,10 @@ class Page extends Component {
     this.setState({ showModal: true });
   }
 
+  isASocietyPage = () => (
+    this.props.location.pathname.split('/')[1] === 'society'
```

Comment:
This function will break if someone ever changed the route to society pages. What if you used `this.props.location.pathname.indexOf('society')` instead?

Reply:
Great call, thanks 👍